src/sparseml/pytorch/utils/exporter.py

```diff
@@ -484,13 +484,18 @@ def export_onnx(
     if "output_names" not in export_kwargs:
         export_kwargs["output_names"] = _get_output_names(out)
 
-    if dynamic_axes == "batch":
-        dynamic_axes = {
-            tensor_name: {0: "batch"}
-            for tensor_name in (
-                export_kwargs["input_names"] + export_kwargs["output_names"]
-            )
-        }
+    if dynamic_axes is not None:
+        warnings.warn(
+            "Setting dynamic_axes is deprecated, now it is always set",
```

Comment:
let's make this warning a bit more specific to show that the value the user set `dynamic_axes` as is no longer respected and we are setting the 0th index across all inputs as dynamic

Reply:
Updated